From aadaff79c00c9dce5f83a8768605e7127e1ea5e3 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Sat, 13 Sep 2014 15:06:39 +0200
Subject: [PATCH] prefs: disable "Use a prediction service to help complete
 searches..."

This autocomplete service would send search queries as soon as you
started typing, even if not yet done typing. Some regard this as an
information leak, others as a waste of bytes sent over the network.
Either reason is fine for me to disable this by default.

Impl. details:
Set suggest_enabled to false by default.
---
 chrome/browser/profiles/profile.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/profiles/profile.cc b/chrome/browser/profiles/profile.cc
index 041981153b20..d49d614a1eca 100644
--- a/chrome/browser/profiles/profile.cc
+++ b/chrome/browser/profiles/profile.cc
@@ -138,7 +138,7 @@ const char Profile::kNoHostedDomainFound[] = "NO_HOSTED_DOMAIN";
 void Profile::RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(
       prefs::kSearchSuggestEnabled,
-      true,
+      false,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
 #if defined(OS_ANDROID)
   registry->RegisterStringPref(
-- 
2.14.3

